helm/cluster/values.schema.json

```diff
@@ -1,175 +1,944 @@
 {
-    "$schema": "http://json-schema.org/schema#",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
Schema is getting really big. Once the development of `cluster` chart stabilizes, the idea is to move parts of the schema to https://github.com/giantswarm/schema, so we can reuse it here and in other charts possibly.